service/login.js

```diff
@@ -0,0 +1,34 @@
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+
+import { WrongCredential } from '../validators/errors';
+import userRepository from '../repository/users';
+
+class LoginService {
+  constructor(userRepository) {
+    this._userRepository = userRepository;
```

Comment:
Again here as well. Since you are not creating a new instance and you are just using what you import there is no need to load the class with this.